package.json

```diff
@@ -74,8 +74,9 @@
     "pretest": "npm run build",
     "posttest": "agadoo internal/index.mjs",
     "prepublishOnly": "npm run lint && PUBLISH=true npm test",
-    "tsd": "tsc -p src/compiler --emitDeclarationOnly && tsc -p src/runtime --emitDeclarationOnly",
-    "lint": "eslint \"{src,test}/**/*.{ts,js}\""
+    "tsd": "tsc -p src/compiler --emitDeclarationOnly --strict false && tsc -p src/runtime --emitDeclarationOnly --strict false",
```

Comment:
Instead of making the default `true` and adding `--strict false` here, could we do the reverse? I.e. leave `strict: true` off, and add `--strict true` in `tsc-strict-error.mjs`. That would allow us to run on one type of strict check at a time in the `tsconfig.json` to allow us to gradually enforce stricter checks

Reply:
I had that thought, but decided to have the flag turn on in the `tsconfig.json`, primarily to have more warning from language server, which in turns shows up in IDE (VSCode) and help contributor to make cleaner code.

After this explanation, if it's still considered better to do the reverse, I'm happy to make the change.